packages/travix-ui-kit/components/calendar/calendar.md

```diff
@@ -30,14 +30,14 @@
           ],
         }}
         minDate="2017-04-01"
-        maxDate="2017-06-01"
+        maxDate="2018-06-01"
         navButtons={{
           days: {
             next: {
-              displayValue: '›'
+              displayValue: <span>›</span>
             },
             previous: {
-              displayValue: '‹'
+              displayValue: <span>‹</span>
```

Comment:
Do you need this change? String should also work with `node` prop type.

Reply:
This change is just to demonstrate possibility to use nodes as 'displayValue' of nav-buttons.